app/controllers/concerns/company_favourites_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CompanyFavouritesController < ApplicationController
+  def create
+    @company = Company.find(params[:company_id])
+    @company_favourite = CompanyFavourite.new
+    @company_favourite.company = @company
+    @company_favourite.developer = current_user.developer
+    if @company_favourite.save
+      redirect_to company_path(@company)
+    else
+      render "company/show"
```

Comment:
If I'm logged in as a company and I try to add another company as a favourite, it crashes the app here. Would be better to remove the favourites button altogether using `current_user`